pkg/client/client.go

```diff
@@ -443,10 +519,10 @@ func (d *DynatraceClient) ListSettings(schemaId string, opts ListSettingsOptions
 			}
 		}
 
-		return nil
+		return len(parsed.Items), len(result), nil
 	}
 
-	err := d.ListPaginated(pathSettingsObjects, params, addToResult)
+	_, err := d.ListPaginated(pathSettingsObjects, params, schemaId, addToResult)
```

Comment:
just noticed this in a discussion about another PR: `ListPaginated` does not need to/should not be exported either. 
Could you please turn this into `listPaginated` when you address the other functions that don't need to be exported? Thanks!

Reply:
Done